### PR TITLE
remove event limits for merges

### PR DESCRIPTION
**Pull request by hufnagel**
Don't want to remove the code completely, but setting the default to 100M events is effectively doing the same thing.

### CODE REVIEW

**Comment by hufnagel** [comment]
test this please

**Comment by hufnagel** [comment]
Interesting, seems the parameters passed to the job splitter in the

WMComponent_t.JobCreator_t.JobCreator_t.JobCreatorTest:testE_TestNonProxySplitting

unit test actually were never successfully passed to the job splitter. This unit test "accidentally" worked and now that I changed the defaults it doesn't anymore.

**Comment by hufnagel** [comment]
Ok, this works now. Ready for review @amaltaro

File: test/python/WMComponent_t/JobCreator_t/JobCreator_t.py [hunk 1]
```diff
@@ -538,6 +538,9 @@ def stuffWMBS(self, workflowURL, name):
             mergeFileset.addFile(file)
             bogusFileset.addFile(file)
 
+#        mergeFileset.markOpen(False)
+#        bogusFileset.markOpen(False)
```
Comment (amaltaro) [inline test/python/WMComponent_t/JobCreator_t/JobCreator_t.py]: Dirk, can you remove these lines please?

**Comment by amaltaro** [comment]
The change looks completely safe, however, do you need to have it in testbed this week and production in the next one? If not, then I'd rather merge it only after the production upgrade to avoid any possible side effect in the validation ...

**Comment by hufnagel** [comment]
There is no hurry with this. Reason I am making the change is since the discussion about event limits came up for the Tier0 and the statement was "we do not need a limit for events per file". We already implemented this for the Tier0, I just want to change the defaults for WMAgent merges to be consistent (eventually).

**Comment by amaltaro** [comment]
Ok.
@ticoann from my side, I'd merge it only after the production upgrade is done (next Tuesday)

**Comment by amaltaro** [comment]
test this please

**Pull request merged**